react-facet/docs/api/hooks.md

```diff
@@ -67,6 +67,7 @@ const Form = ({ onSubmit, initialValue }: Props) => {
 ## `useFacetCallback`
 
 The `useFacetCallback` hook is similar to React’s `useCallback` in that it allows you to create a memoized callback that will only be updated if some of the explicit dependencies change. On top of that, `useFacetCallback` allows you to pass one or more Facets and get the current values of those facets in the callback body.
+Like `useCallback`, `useFacetCallback` will update the reference if a value in the dependency array change. However, it will not update the reference if a Facet change.
```

Comment:
```suggestion

Like `useCallback`, `useFacetCallback` will update the reference if a value in the dependency array change. However, it will not update the reference if a Facet change.
```